runner/src/server/plugins/engine/components/helpers.ts

```diff
@@ -90,3 +91,34 @@ export const addClassOptionIfNone = (
     options.classes = className;
   }
 };
+
+export function getVarsFromContent(content: string) {
```

Comment:
I think we should probably just use njks and renderString in this case. 

We should also add an environment variable `ALLOW_USER_TEMPLATES`. 

If true - we'll use njks and render. It looks like it could be pretty easy to inject script tags otherwise